src/components/ErrorForm/ErrorForm.jsx

```diff
@@ -1,42 +1,29 @@
-import React from 'react'
+import React, { useState } from 'react'
 import formStyle from './errorForm.module.css'
 
-class ErrorForm extends React.Component {
-  constructor(props) {
-    super(props)
+export default function ErrorForm() {
+  const [emailInputInvalid, setEmailInputInvalid] = useState(false)
 
-    this.state = {
-      emailInputInvalid: false,
-    }
-
-    this.fakeSubmit = this.fakeSubmit.bind(this)
-  }
-
-  fakeSubmit(e) {
+  function fakeSubmit(e) {
     e.preventDefault()
     if (e.currentTarget.value === '') {
-      this.setState({ emailInputInvalid: true })
+      setEmailInputInvalid(true)
     }
```

Comment:
It's just a style preference thing, but I think it would be nice to change this to an arrow function while we're here -

`const fakeSubmit = e => {
    if (e.currentTarget.value === '') {
        setEmailInputInvalid(true)
    }
}`

Reply:
Will do! I'll make sure to add that in everywhere. I only used the other syntax because I noticed your previously-refactored components use `export default function ComponentName` and thought that was the preferred syntax :)

Reply:
thanks so much! Yeah I really do need to put together a style guide... and then try and stick to it haha